src/frontend/js/index.js

```diff
@@ -884,7 +767,7 @@ function getUnit(meta, domain, variable, percent=false){
         if (domain == "od_matrices") {
             return meta.od_matrices[variable].unit
         } else {
-            return meta.links[variable].unit
+            throw "unreachable"
```

Comment:
```suggestion
            throw new Error("unreachable")
```
Throwing strings is a trap, you don't get a backtrace